src/opnsense/mvc/app/controllers/OPNsense/Base/ApiControllerBase.php

```diff
@@ -91,6 +91,7 @@ public function initialize()
     {
         // disable view processing
         set_error_handler(array($this, 'APIErrorHandler'));
+        self::setLocale();
```

Comment:
Eventually it might be better to move both setLocales() to ControllerRoot, for now this is good enough.